gui/src/2D/controls/container.ts

```diff
@@ -387,11 +409,22 @@ export class Container extends Control {
 
     /** @hidden */
     public _draw(context: CanvasRenderingContext2D, invalidatedRectangle?: Measure): void {
+        let contextToDrawTo: CanvasRenderingContext2D = (this._intermediateTexture) ? this._intermediateTexture.getContext() : context;

```

Comment:
nit: `const` instead of `let`

Reply:
Changed this to work with the above point